src/test/ui/structs/struct-fn-in-definition.stderr

```diff
@@ -33,6 +33,7 @@ LL |     fn foo() {}
    |
    = help: unlike in C++, Java, and C#, functions are declared in `impl` blocks
    = help: see https://doc.rust-lang.org/book/ch05-03-method-syntax.html for more information
+   = help: enum variants can be `Variant` or `Variant = <integer>` or `Variant(Type)` or `Variant { field: Type }`
```

Comment:
The extra help message seems a little unexpected to me. It's technically true, but I don't think it's really the problem the user is running into here. I'm not sure it's worth the extra logic to differentiate this case though?

Reply:
changed the suggestion message to:
`enum variants can be `Variant`, `Variant = <integer>`, `Variant(Type, ..., TypeN)` or `Variant { fields: Types }``